### PR TITLE
add waterfall static viz e2e specs

### PR DESCRIPTION
**Pull request by alxnddr**
### Description

Add waterfall static viz e2e tests that check the following states:
- Show total is not specified: should show total by default
- Show total is `false`
- Show total is `true`

### How to verify
- Check waterfall charts in Percy look as expected

### CODE REVIEW

**Comment by alxnddr** [comment]
@metabase-bot run visual tests

File: frontend/test/__support__/e2e/commands/api/composite/createQuestionAndAddToDashboard.js [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
   "createQuestionAndAddToDashboard",
   (query, dashboardId) => {
-    return cy.createQuestion(query).then(response => {
-      cy.request("POST", `/api/dashboard/${dashboardId}/cards`, {
-        cardId: response.body.id,
-      });
+    return (query.native
+      ? cy.createNativeQuestion(query)
+      : cy.createQuestion(query)
+    ).then(response => {
```
Comment (kulyk) [inline frontend/test/__support__/e2e/commands/api/composite/createQuestionAndAddToDashboard.js:7]: Doesn't[ `createQuestion` command](https://github.com/metabase/metabase/blob/master/frontend/test/__support__/e2e/commands/api/question.js) allow to create both GUI and native questions?

File: frontend/test/metabase-visual/static-visualizations/waterfall.cy.spec.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+describe("static visualizations", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+    setupSMTP();
+  });
+
+  it(`waterfall chart`, () => {
+    const dashboardName = `Waterfall charts dashboard`;
```
Comment (kulyk) [inline frontend/test/metabase-visual/static-visualizations/waterfall.cy.spec.js:19]: nip: I believe it makes sense to use backticks only for template strings. We should maybe enable a linter rule for that

File: frontend/test/__support__/e2e/commands/api/composite/createQuestionAndAddToDashboard.js (hunk 1 above)
Comment (alxnddr) [reply frontend/test/__support__/e2e/commands/api/composite/createQuestionAndAddToDashboard.js:7]: @kulyk, unfortunately, it's not. Maybe we don't need two different commands 🤔

File: frontend/test/metabase-visual/static-visualizations/waterfall.cy.spec.js (hunk 2 above)
Comment (alxnddr) [reply frontend/test/metabase-visual/static-visualizations/waterfall.cy.spec.js]: Thanks, fixed. I'll take a look at the linter rule

**Review by kulyk — APPROVED** [review]

File: frontend/test/__support__/e2e/commands/api/composite/createQuestionAndAddToDashboard.js (hunk 1 above)
Comment (alxnddr) [reply frontend/test/__support__/e2e/commands/api/composite/createQuestionAndAddToDashboard.js:7]: @nemanjaglumac hmm, I got an error, maybe I misused it. Do we need different commands then?

**Pull request merged**